soh/soh/Enhancements/gameplaystats.cpp

```diff
@@ -47,7 +165,9 @@ void DisplayTimeHHMMSS(uint32_t timeInTenthsOfSeconds, const char* text, ImVec4
 
     ImGui::PushStyleColor(ImGuiCol_Text, color);
 
-    ImGui::Text(text);
+    std::string text2(text);
+    std::string padded = fmt::format("{:<40}", text2);
```

Comment:
The function would've broke if we tried to format the `const char *`. Changing the parameter type to `std::string` doesn't do anything to break since I'm guessing that `std::string` can quietly deal with `char[]` with no issues.